vignettes/Antimalarial_Resistance.Rmd

```diff
@@ -0,0 +1,266 @@
+---
+title: "Antimalarial Resistance"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Antimalarial Resistance}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+
+```{r setup}
+
+# Load the requisite packages:
+#library(malariasimulation)
+
+# Use devtools to load the updated malariasimulation:
+devtools::load_all("C:/Users/trb216/OneDrive - Imperial College London/Desktop/malariasimulation/malariasimulation.Rproj")
```

Comment:
To get the automated checks to pass!

```suggestion
# Load the requisite packages:
library(malariasimulation)
```

Reply:
Good spot - removed the `load_all()` call, uncommented `library(malariasimulation)` and confirmed vignette can build!